ruby_programming/intermediate_ruby/project_file_io.md

```diff
@@ -21,6 +21,7 @@ Send us your solution so we can show others! Submit a link to the Github [repo](
   <summary> Show Student Solutions </summary>
 
 * Add your solution below this line!
+* [Andrew Huntington's Solution] (https://github.com/AndrewHuntington/hangman)
```

Comment:
You have a space here. Please make sure you're following the proper formatting, there are examples on each page on what you should do. I'll fix this one for you, but be aware moving forward.